env/pretty_print.bash

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+pretty_echo() {
+    local type=$1
+    shift
+    local message="$@"
+
+    local red='\033[0;31m'
+    local yellow='\033[1;33m'
+    local green='\033[0;32m'
+    local blue='\033[0;34m'
+    local cyan='\033[0;36m'
+    local nc='\033[0m'
+    local bold='\033[1m'
+
+    case "${type,,}" in
```

Comment:
Maybe add a \t between the label and message for consistent spacing in outputs?

Reply:
looks better now, done

Reply:
![image](https://github.com/user-attachments/assets/cb838935-a9e7-45d7-b08a-1968e7df4321)